pandas/io/sql.py

```diff
@@ -432,7 +432,9 @@ def to_sql(frame, name, con, flavor=None, schema=None, if_exists='fail',
         library.
         If a DBAPI2 object, only sqlite3 is supported.
     flavor : 'sqlite', default None
-        DEPRECATED: this parameter will be removed in a future version
+        .. deprecated:: 0.19.0
+           'sqlite' is the only supported option if SQLAlchemy is not
+           installed.
```

Comment:
'installed' is not fully correct, as you can use the sqlite backend also when sqlalchemy is installed. So I would make 'used' of it.

(the same for all the other occurrences of this one both in this file as in generic.py)